### DISCUSSION TITLE
What settings would you recommend for someone looking to increase the crime in their city to realistic-ish levels?

### DISCUSSION THREAD

**Discussion by stripeyspacey** [Roll-W/CSL2-CrimeRemover#4] [Q&A]
I can't be the only one that noticed that in the vanilla game, your prisons are always, _always_ empty! Jails too. So I got this mod specifically for the possibility of having _**more**_ crime added.. but I haven't seen any difference with the increases I've made. 

Are there any recommended settings for achieving this? I don't want "Vice City" lol, but I don't want Mr. Roger's Neighborhood either :)

**Comment by roll-w** [comment]
The problem is that the vanilla game has two factors that can affect crime, one is the building crime, which is the one you can set in the mod; and the other one is criminals, people who actually commit crimes, which currently cannot be set in the mod. It is only when these two factors are not zero that someone commits a crime and is further arrested.

There are also some conditions in game to choose who will be the criminal, and since this mod is designed to remove crime, I didn't mess with it too much. Basically, setting the crime rate in the mod will increase the possibility that the game choose who to be the criminal, but there is no direct way to set the criminals in the mod.
